assignment.hs

```diff
@@ -94,15 +94,17 @@ reciprocal x = . . .
 
 eulerSum n = sqrt $ 6 * sumOfFirst n series
 
+-- It appears to converge on pi!
+-- Passing 10000000 gave 3.1415925580959025 which is accurate to 6 decimal places, not too shabby.
```

Comment:
I wonder sometimes how different our mathematics would be if we'd started not with counting numbers but with using base pi. Maybe somewhere in the universe there's a species of circular folks who did just that